modules/client/src/validation/appProposals.ts

```diff
@@ -107,6 +107,7 @@ export const validateLinkedTransferApp = async (
 };
 
 export const appProposalValidation: ProposalValidator = {
+  SimpleLinkedTransferApp: validateLinkedTransferApp,
   SimpleTransferApp: validateSimpleTransferApp,
   SimpleTwoPartySwapApp: validateSwapApp,
   UnidirectionalLinkedTransferApp: validateLinkedTransferApp,
```

Comment:
Apps like `SimpleLinkedTransferApp` are supposed to replace apps like `UnidirectionalLinkedTransferApp` right? So eventually all the unidirectional ones will be removed?

Reply:
Yes, this does replace Unidirectional, we should take that out completely.